docs/_docs/extensions-and-integrations/spring/spring-tx.adoc

```diff
@@ -0,0 +1,471 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Apache Ignite With Spring Transactions
+
+== Overview
+
+http://docs.spring.io/spring/docs/current/spring-framework-reference/html/transaction.html[Spring Transactions, window="_blank"]
+abstraction allows you to use declarative transaction management to concentrate on business logic rather than
+transaction life-cycle.
```

Comment:
```suggestion
link:http://docs.spring.io/spring/docs/current/spring-framework-reference/html/transaction.html[Spring transactions,window=_blank] abstraction allows you to use a declarative transaction management to concentrate on business logic rather than transaction life-cycle.
```